config/permissions.php

```diff
@@ -82,6 +82,9 @@
         '/services/getUserCityHistoriesService.php', '/services/getUserJourneyHistoriesService.php',
         '/services/getTodayAreaUsersService.php', '/services/getProjectUsersService.php',
         '/services/getProjectCustomersService.php'),
+    'manager' => array(
+        '/projectsEvaluation.php', '/usersEvaluation.php', '/projectsSummary.php',
+    ),
```

Comment:
You need to check which are the services related to those report screens and prevent access to them too.